justfile

```diff
@@ -108,5 +108,5 @@ fix: devenv
 
 
 # Run JupyterLab
-run: devenv
-    opensafely jupyter
+run *ARGS: devenv
```

Comment:
this is more of a question than a comment. I assume you did this so we can specify some of these in the just command?  

```
usage: opensafely jupyter [-h] [--directory DIRECTORY] [--name NAME] [--no-browser] [--port PORT]

options:
  -h, --help            show this help message and exit
  --directory DIRECTORY, -d DIRECTORY
                        Directory to run the jupyter server in (default is current dir)
  --name NAME           Name of docker image (defaults to use directory name)
  --no-browser, -n      Do not attempt to open a browser
  --port PORT, -p PORT  Port to run on
```

Reply:
Yes, that's correct. Specifically, I wanted to pass `--no-browser` to stop JupyterLab taking control. I often open several environments at the same time; some will start a JupyterLab server. It's distracting if each then opens a new tab in my browser.